### ISSUE TITLE
Major 3.3 Testnet Issues

### ISSUE THREAD

**Issue by lk86** [MinaProtocol/mina#6019]
This issue acts as a list / tracker of the major issues folks are experiencing in the v3.3 testnet. Follow this issue and the linked issues below that affect you to get further status updates on fixing them.

#5999 
#6000 
#6001 
#6002 
#6003 

If you do not think a crash you are experiencing is covered by one of the above issues, create an issue and attach the log from your coda daemon and link the new issue in a comment on this thread.

**Comment by gaia** [comment]
https://github.com/CodaProtocol/coda/issues/5991

**Comment by garethtdavies** [comment]
Lots of #6071 

I've linked a lot to #6049 and `failed_to_close_pipe` errors

**Comment by garethtdavies** [comment]
#6094 is `An empty epoch is detected!` error that most users are now experiencing.

**Comment by mengsuenyan** [comment]
[6094](https://github.com/MinaProtocol/mina/issues/6094), I've have the same problem when running a locally compiled program, but running the official program is normal.


`
lzj@lzj:~/mina/test/mina$ export CODA_LIBP2P_HELPER_PATH=./src/app/libp2p_helper/result/bin/libp2p_helper 
lzj@lzj:~/mina/test/mina$ ./_build/default/src/app/cli/src/coda.exe daemon -peer-list-file ~/peers.txt -block-producer-key ~/keys/my-wallet -block-producer-password "passwd" -generate-genesis-proof true -file-log-level trace -seed
2020-12-22 02:42:54 UTC [Info] Coda daemon is booting up; built with commit "41db20648b566908881a17c8b0248d05f517f25a" on branch "master"
2020-12-22 02:42:54 UTC [Info] Removing lockfile for terminated process
2020-12-22 02:42:54 UTC [Info] Booting may take several seconds, please wait
2020-12-22 02:42:54 UTC [Info] Reading configuration files [ "/home/lzj/.coda-config/daemon.json" ]
2020-12-22 02:42:54 UTC [Warn] Could not read configuration from "/home/lzj/.coda-config/daemon.json": $error
	error: {
  "exn_name": "Async_kernel__Monitor.Error_",
  "exn":
    "(monitor.ml.Error\n  (Unix.Unix_error \"No such file or directory\" open\n    \"((filename /home/lzj/.coda-config/daemon.json) (mode (O_RDONLY O_CLOEXEC)) (perm 0o0))\")\n  (\"Raised at file \\\"src/result.ml\\\", line 158, characters 17-26\"\n    \"Called from file \\\"src/deferred1.ml\\\", line 17, characters 40-45\"\n    \"Called from file \\\"src/job_queue.ml\\\" (inlined), line 131, characters 2-5\"\n    \"Called from file \\\"src/job_queue.ml\\\", line 171, characters 6-47\"\n    \"Caught by monitor try_with_or_error\"))"
}
2020-12-22 02:42:54 UTC [Info] Initializing with runtime configuration. Ledger name: null
2020-12-22 02:42:54 UTC [Info] Using the compiled constraint constants
2020-12-22 02:42:54 UTC [Info] Found genesis proof file at $path
	path: "/home/lzj/.coda-config/genesis/genesis_proof_4be1a4ac6655c08582838def2c3292f43859a928c987debc094c8dce7a403275"
2020-12-22 02:42:54 UTC [Info] Loaded ledger from $ledger_file and genesis proof from $proof_file
	ledger_file: "/home/lzj/.coda-config/genesis/genesis_ledger_test_3536ac15fbf80f54d4dbf4b145d30244ec15c3bba4c0d4a88dfb8a8f7e57d908.tar.gz"
	proof_file: "/home/lzj/.coda-config/genesis/genesis_proof_4be1a4ac6655c08582838def2c3292f43859a928c987debc094c8dce7a403275"
2020-12-22 02:42:55 UTC [Info] Starting node as a seed node
2020-12-22 02:42:55 UTC [Info] Setting current protocol version to "1.1.0" from compile-time config
2020-12-22 02:42:55 UTC [Info] Daemon started process of kind "Prover" with pid 5385
2020-12-22 02:42:55 UTC [Info] Registering async shutdown handler: "Close transition frontier, if exists"
2020-12-22 02:42:55 UTC [Info] libp2p peer ID this session is $peer_id
	peer_id: "12D3KooWGwVfULQbN4AqGSgHwa1jZzWLW2jfQJJEpyNbEt9pYYQj"
2020-12-22 02:42:56 UTC [Info] Coda daemon is connecting
2020-12-22 02:42:56 UTC [Info] Initializing plugins
2020-12-22 02:42:56 UTC [Info] Pausing block production while bootstrapping
2020-12-22 02:42:56 UTC [Info] Attempted to turn on snark worker, but snark worker key is set to none
2020-12-22 02:42:56 UTC [Info] Daemon ready. Clients can now connect
2020-12-22 02:42:56 UTC [Info] Created GraphQL server at: http://localhost:3085/graphql
2020-12-22 02:43:56 UTC [Info] Will start initialization without connecting with too any peers
2020-12-22 02:43:56 UTC [Info] Fast forward successful
2020-12-22 02:43:56 UTC [Info] Loaded full frontier and extensions
2020-12-22 02:43:56 UTC [Info] Successfully loaded frontier
2020-12-22 02:43:56 UTC [Info] Requesting peers for their best tip to do initialization
2020-12-22 02:43:56 UTC [Info] Successfully loaded frontier, but failed to download best tip from network; starting participation
2020-12-22 02:43:56 UTC [Info] Starting transition frontier controller phase
2020-12-22 02:43:56 UTC [Info] Coda daemon is connecting
2020-12-22 02:43:56 UTC [Info] Updating new available work took 0.014543533325195312 ms
2020-12-22 02:43:56 UTC [Info] Determining next slot to produce block
2020-12-22 02:43:56 UTC [Fatal] An empty epoch is detected! This could be caused by the following reasons: system time is out of sync with protocol state time; or internet connection is down or unstable; or the testnet has crashed. If it is the first case, please setup NTP. If it is the second case, please check the internet connection. If it is the last case, in our current version of testnet this is unrecoverable, but we will fix it in future versions once the planned change to consensus is finished.
`
